src/Build/BuildCop/API/BuildAnalyzerConfiguration.cs

```diff
@@ -42,4 +45,37 @@ public class BuildAnalyzerConfiguration
     /// If some rules are enabled and some are not, the analyzer will be run and reports will be post-filtered.
     /// </summary>
     public bool? IsEnabled { get; internal init; }
+
+    public static BuildAnalyzerConfiguration Create(Dictionary<string, string> configDictionary)
+    {
+        return new()
+        {
+            EvaluationAnalysisScope = TryExtractValue("EvaluationAnalysisScope", configDictionary, out EvaluationAnalysisScope evaluationAnalysisScope) ? evaluationAnalysisScope : null,
+            Severity = TryExtractValue("severity", configDictionary, out BuildAnalyzerResultSeverity severity) ? severity : null,
+            IsEnabled = TryExtractValue("IsEnabled", configDictionary, out bool test) ? test : null,
```

Comment:
We should probably define those strings as constants.
Other option is to use `nameof` to get the same string as the property names in the target class.

Reply:
Seems like editrconfig parsing is flipping the keys to lowercase - should we have the extracted keys here in lowercase as well?

Reply:
In first local test scenarios only  the Severity was working fine without respsecting IsEnabled property. Updated to usage of the nameof